input/pagecontent/group-disease.md

```diff
@@ -42,6 +42,13 @@ Several widely-used terminologies in the cancer domain, including ICD-O-3 and AJ
 
 Under the [Fair Use doctrine](https://www.copyright.gov/fair-use/more-info.html), the IG includes examples illustrating mCODE's representation of cancer diagnoses and AJCC staging values for the purposes of technical implementation guidance to FHIR developers.
 
+### Representing non-TNM Staging Information
+
+Not all cancer types are staged with a TNM-based staging system. Hematological cancers like leukemias, multiple myeloma, and some lymphomas.  Some specialized solid tumors like gynecologic tumors are staged using the FIGO (International Federation of Gynecology and Obstetrics) staging system. Staging systems in this category include Rai, Binet, Cottswold
+Support for non-TNM staging systems in mCODE is limited to [CancerStageGroup]. Use NCI thesaurus codes for specifying cancer staging systems which do not have an equivalent  SNOMED code. 
+
+Prognostic factors related to the cancer stage group can be specified with the Observation._derivedFrom_ element. For example, a hemoglobin lab result which was evaluated in the  staging of chronic lymphocytic leukemia using the Binet staging system can be referenced under Observation.derivedFrom element. The mCODE example [`binet-stage-group-A`](Observation-binet-stage-group-A.html) illustrates how this could be represented.
```

Comment:
```suggestion
Prognostic factors related to the cancer stage group can be specified with the `Observation.derivedFrom` element. For example, a hemoglobin lab result which was evaluated in the  staging of chronic lymphocytic leukemia using the Binet staging system can be referenced under `Observation.derivedFrom` element. [This example](Observation-binet-stage-group-A.html) illustrates how this could be represented.
```